core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -345,7 +350,7 @@ private PutResult putAttempt(PageId pageId, byte[] page, CacheContext cacheConte
           pageStoreDir.getPageStore().put(pageId, page, cacheContext.isTemporary());
           // Bytes written to the cache
           MetricsSystem.meter(MetricKey.CLIENT_CACHE_BYTES_WRITTEN_CACHE.getName())
-              .mark(page.length);
+              .mark(page.position());
```

Comment:
This should be something like 

```java
int contentSize = page.remaining();
pageStore.put(page);
meter.mark(contentSize);
```

because the buffer's position may be not 0 at the beginning, therefore, buffer.position() may not be equal to buffer.remaining().

Reply:
done